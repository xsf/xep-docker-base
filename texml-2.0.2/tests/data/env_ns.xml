<TeXML xmlns="http://getfo.sourceforge.net/texml/ns1">
<env name="document">
text
<env name="st" begin="s" end="e">ET phone home!</env>
text
<env name="test" nl2="0">
	Test whitespaces in an environment
</env>
<env name="test">
	<parm>a</parm>
	<opt>b</opt>
	*
	<parm>c</parm>
	Test extended environment
	<opt>Incorrect, but interesting. Also see '*' above</opt>
</env>
</env>
</TeXML>
