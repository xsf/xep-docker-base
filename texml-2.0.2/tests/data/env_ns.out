\begin{document}
text
\s{st}
ET phone home!
\e{st}
text
\begin{test} Test whitespaces in an environment
\end{test}
\begin{test}{a}[b]
*{c} Test extended environment[Incorrect, but interesting. Also see '*' above]
\end{test}
\end{document}
